go.mod

```diff
@@ -11,6 +11,7 @@ require (
 	github.com/onflow/cadence v1.7.0
 	github.com/onflow/crypto v0.25.3
 	github.com/onflow/flow/protobuf/go/flow v0.4.11
+	github.com/onflow/go-ethereum v1.16.2
```

Comment:
The fork should not be re-introduced, use upstream `"github.com/ethereum/go-ethereum/rlp"`

```suggestion
```